services/venice-controller/src/main/java/com/linkedin/venice/controller/multitaskscheduler/StoreMigrationManager.java

```diff
@@ -73,6 +76,8 @@ public static StoreMigrationManager createStoreMigrationManager(
     this.delayInSeconds = delayInSeconds;
     this.migrationRecords = new ConcurrentHashMap<>();
     this.migrationTasks = new ConcurrentHashMap<>();
+    // Create a redundancy filter with ~1M bit slots (~128KB) and a 20-minute suppression window for duplicate logs
+    this.filter = new RedundantExceptionFilter(1024 * 1024, TimeUnit.MINUTES.toMillis(20));
```

Comment:
How many unique messages you could foresee in 20 minutes? Is it at store level, or partition level, or no patterns at all?